### PR TITLE
Add version command

### PR DESCRIPTION
**Pull request by ForTheReallys**

### CODE REVIEW

File: Doc/cmus.txt [hunk 1]
```diff
@@ -632,6 +632,9 @@ update-cache [-f]
 	-f
 		Update all files. Same as quit, rm -f $XDG_CONFIG_HOME/cmus/cache, start cmus.
 
+version
+	print version
```
Comment (flyingmutant) [inline Doc/cmus.txt]: `Print the version information`.

**Comment by flyingmutant** [comment]
Sorry for the delayed review! Looks good, but I think documentation can be made a bit better.

File: Doc/cmus.txt [hunk 2]
```diff
@@ -632,6 +632,9 @@ update-cache [-f]
 	-f
 		Update all files. Same as quit, rm -f $XDG_CONFIG_HOME/cmus/cache, start cmus.
 
+version
+	print the version information
```
Comment (flyingmutant) [inline Doc/cmus.txt]: Sorry to nitpick, but all descriptions should look like full sentences: start with a capital letter and end with a period.

**Comment by ForTheReallys** [comment]
No problem.  Thanks!

**Pull request merged**